src/handbook/development/contributing.md

```diff
@@ -61,4 +61,9 @@ When merging a PR, you should choose the "Merge pull request" option. There is n
 
 When conducting a PR review, if you are the last (or only) reviewer and all reviews (including your own) are approvals, unless there is a comment from the author stating otherwise, you are free to conduct the merge. Otherwise, leave the merge to the author of the PR, or a future reviewer.
 
+For a comprehensive review of the Pull Request, utilize the designated FlowFuse pre-staging environment. As of the composition of this document, the pre-staging verification is only available for the primary [FlowFuse NPM package](https://github.com/FlowFuse/flowfuse).
+To validate changes within the pre-staging environment, apply the deploy:pr label to the PR. The PR will then undergo automatic deployment to the pre-staging environment, and the deployment link will be appended to the associated workflow summary. The label should remain until the PR is merged.
```

Comment:
```suggestion
To validate changes within the pre-staging environment, apply the `deploy:pr` label to the PR. The PR will then undergo automatic deployment to the pre-staging environment, and the deployment link will be appended to the associated workflow summary. The label should remain until the PR is merged.
```

Reply:
Can we also get a bit more detail on where to find the link (maybe a screenshot) - given these instructions, and [this example PR](https://github.com/FlowFuse/flowfuse/pull/3652#discussion_r1542676749) - I cannot find the link